hw/Makefile

```diff
@@ -47,6 +47,7 @@ build:
 	vsim -c -do 'source compile.tcl; quit'
 
 lib:
+	@touch modelsim.ini
```

Comment:
Why do you need to touch `modelsim.ini`?

Reply:
Because the next thing we do is a chmod on that file, so we need to make sure it exists

Reply:
The issue actually shows up because `modelsim.ini` is automatically generated during the `vmap` command. We can, thus, fix the issue by swapping the `@vmap work work` and the `chmod +w modelsim.ini` commands in the [lib](https://github.com/pulp-platform/redmule/blob/master/hw/Makefile#L49) rule.

Reply:
I will merge the PR and fix it.